plugins/modify-secret.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: modify-secret
+spec:
+  version: v0.0.4
+  platforms:
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: https://github.com/rajatjindal/kubectl-modify-secret/releases/download/v0.0.4/darwin-amd64-v0.0.4.tar.gz
+    sha256: 388922fd752889f61598cb72240c62a8bc308146b827be8f9b87c3618251557a
+    files:
+    - from: "*"
+      to: "."
+    bin: kubectl-modify-secret
+  - selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+    uri: https://github.com/rajatjindal/kubectl-modify-secret/releases/download/v0.0.4/linux-amd64-v0.0.4.tar.gz
+    sha256: a0c9928d25deb587b7c97a6cbab654aae045fa77c08ef3b420062bb86cf95540
+    files:
+    - from: "*"
+      to: "."
+    bin: kubectl-modify-secret
+  shortDescription: modify secret with implicit base64 translations
+  caveats: |
+    Usage:
+      kubectl modify-secret secret-name -n kube-system
+    
+    Read the documentation at:
+      https://github.com/rajatjindal/kubectl-modify-secret
+  description: |
+    Usage:
+      kubectl modify-secret secret-name -n kube-system
+
+      This plugin fetches the given secret manifest from the cluster, decodes the payload, opens an editor to make changes, and applies the modified manifest when done.
```

Comment:
Can you break this line so that it wraps at 80 characters? This will ensure that it prints nicely for people with small terminal width (80 chars seems to be a common lower limit).